content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Use a user namespace with a stateless pod
+reviewers:
+content_type: task
+weight: 160
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+This page shows how to configure a user namespace for stateless pods. This
+allows to isolate the user running inside the container from the one in the
+host. A process running as root in a container can run as a different (non-root)
+user in the host.
+
+You can use this feature to reduce the damage a compromised container can do to
+the host or other pods in the same node. There are [several security
+vulnerabilities][KEP-vulns] rated HIGH and CRITICAL that were not explitable
+when user namespaces is active. It is expected user namespace will mitigate some
+future vulnerabilities too.
+
+[KEP-vulns]: https://github.com/kubernetes/enhancements/tree/217d790720c5aef09b8bd4d6ca96284a0affe6c2/keps/sig-node/127-user-namespaces#motivation
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Furthermore, you also need support in your container runtime to use this feature
+with Kubernetes stateless pods:
+
+* CRI-O: v1.25 has support for user namespaces.
+
+* containerd: support is planned for the 1.7 release. See containerd
+  issue [#7063][containerd-userns-issue] for more details.
+
+Support for this in [cri-dockerd is not planned][CRI-dockerd-issue] yet.
```

Comment:
Please [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future), and especially avoid making statements about the future of other projects. Maybe just mention that CRI-O has support.

Reply:
@sftim but these links are very relevant for users. I can change the wording, but I really really really want to link these issue from some part of the k8s documentation. What do you recommend?